onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -1319,6 +1319,41 @@ def sample_inputs_scaled_dot_product_flash_attention(
     yield from samples
 
 
+def sample_inputs__scaled_dot_product_efficient_attention(
+    op_info, device, dtype, requires_grad, **kwargs
+):
+    del op_info
+    del kwargs
+
+    make = opinfo_core.partial(
+        opinfo_core.make_tensor, device=device, dtype=dtype, requires_grad=requires_grad
```

Comment:
You may also control device here

Reply:
I enabled the whole CUDA tests if it's needed by the test.